src/corecel/grid/GridTypes.hh

```diff
@@ -0,0 +1,55 @@
+//------------------------------- -*- C++ -*- -------------------------------//
+// Copyright Celeritas contributors: see top-level COPYRIGHT file for details
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file corecel/grid/GridTypes.hh
```

Comment:
There's also `InterpolationType` in celeritas/Types.hh (which might overlap somewhat with `Interp`)

Reply:
Hmm I'm going to leave that in place for now I think.